eosio/deployment/BancorX/BancorX.cpp

```diff
@@ -0,0 +1,289 @@
+
```

Comment:
This and the hpp should be diffed against the version that Bancor is (still) running.

Reply:
this is the commit you are looking for:
https://github.com/deweb-io/token/commit/0444cb5a8aa7407df32af64359551bae468aefd7